war/src/main/js/widgets/config/model/ConfigTableMetaData.js

```diff
@@ -47,6 +47,16 @@ exports.fromConfigTable = function(configTable) {
     // The first set of rows don't have a 'section-header-row', so we manufacture one,
     // calling it a "General" section. We do this by marking the first row in the table.
     // See the next block of code.
+    
+    var generalRow = $('<tr class="section-header-row insert first" title="General"><td colspan="4"><div class="section-header"><a class="section-anchor">#</a>General</div></td></tr>');
+    if(!firstRow.hasClass('section-header-row')){
+       firstRow.before(generalRow);
+       firstRow = configTableMetadata.getFirstRow();
+       var newArray = $.makeArray(topRows);
+       newArray.unshift(generalRow[0]);
+       topRows = $(newArray);
```

Comment:
So with the tabs, we are hiding that first line and in the case of the 'general' tab, that first line has the name input in it. it is sort of ok to hide that guy, cause it was already set a screen earlier, likely doesn't need to be edited now, but to me, it seems like feature change decision shouldn't happen here. The shortest distance to making tabs without headers fit the pattern of the other tab sections seemed to me to be to add the header and continue on as normal.